src/SprykerEco/Shared/FactFinderSdk/Transfer/fact_finder_sdk.transfer.xml

```diff
@@ -1,31 +1,37 @@
 <?xml version="1.0"?>
-<transfer
-    xmlns="http://xsd.spryker.com"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://xsd.spryker.com ../../src/Spryker/TransferObjectDefinitionSchema.xsd"
->
+<transfers xmlns="spryker:transfer-01"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="spryker:transfer-01 http://static.spryker.com/transfer-01.xsd">
+
+    <transfer name="CategoryDataFeed">
+    </transfer>
+
+    <transfer name="Currency">
+        <property name="name" type="string"/>
+        <property name="idCurrency" type="int"/>
+    </transfer>
 
     <transfer name="FactFinderSdkSearchRequest">
-        <property name="request" type="array"/>
+        <property name="request" type="array" singular="request"/>
     </transfer>
 
     <transfer name="FactFinderSdkSearchResponse">
         <property name="campaignIterator" type="FactFinderSdkDataCampaignIterator"/>
         <property name="afterSearchNavigation" type="FactFinderSdkDataAfterSearchNavigation"/>
-        <property name="breadCrumbs" type="FactFinderSdkDataBreadCrumb[]"/>
+        <property name="breadCrumbs" type="FactFinderSdkDataBreadCrumb[]" singular="breadCrumb"/>
```

Comment:
- It looks like a BC break, for compatibility use `breadCrumbs`.
- Check other places.

Reply:
Fixed.